contracts/stellar-axelar-example/src/contract.rs

```diff
@@ -27,7 +25,6 @@ pub enum AxelarExampleError {
     InvalidAmount = 3,
 }
 
-#[contractimpl]
```

Comment:
This should not have had the annotation, because the `CustomAxelarExecutable` shouldn't be part of the contract API